var/spack/repos/builtin/packages/py-smartypants/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PySmartypants(PythonPackage):
+    """smartypants is a Python fork of SmartyPants."""
+
+    homepage = "https://github.com/leohemsted/smartypants.py"
+
+    # PyPI only has the wheel
+    url      = "https://github.com/leohemsted/smartypants.py/archive/refs/tags/v2.0.1.tar.gz"
+
+    version('2.0.1', sha256='b98191911ff3b4144ef8ad53e776a2d0ad24bd508a905c6ce523597c40022773')
```

Comment:
Missing a build dep on setuptools

Reply:
I did not see setuptools in the setup.py file. Hunting around I see this in the spack docs:

    If there is no pyproject.toml file and only a setup.py or setup.cfg file, you can assume that the project uses setuptools. 

I will have to remember that.

Reply:
The specific reason why setuptools is needed is documented here: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#distutils